BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -364,3 +343,81 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     }
   }
 }
+
+extension WalletViewController {
+  
+  func startNotificationObserver() {
+    // Stopping as a precaution
+    stopNotificationObserver()
```

Comment:
Not necessary if your worry is that you'll a second observer, and since you overwrite `currentNotification` in `loadNextNotification` that's not an issue either